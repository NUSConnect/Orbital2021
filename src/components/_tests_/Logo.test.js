import * as React from "react";
import Logo from "../Logo";
import renderer from "react-test-renderer";

it(`renders correctly`, () => {
    const tree = renderer.create(<Logo />);
    expect(tree).toMatchSnapshot();
});