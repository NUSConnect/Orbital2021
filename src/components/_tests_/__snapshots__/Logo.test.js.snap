// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={1}
  style={
    Object {
      "height": 110,
      "marginBottom": 8,
      "width": 110,
    }
  }
/>
`;
